packages/backend/src/controllers/moodle/schemas/moodle.schema.ts

```diff
@@ -6,52 +7,52 @@ type Timestamp = number;
 type Milliseconds = number;
 
 interface IMoodleSchema extends Document {
-    [_id: string]: any;
-    lastFetch: Timestamp;
-    refreshRate: Milliseconds;
-    nextFetch: Timestamp;
+  [_id: string]: any;
+  lastFetch: Timestamp;
+  refreshRate: Milliseconds;
+  nextFetch: Timestamp;
 }
 
 interface IModelMoodleSchema extends Model<IMoodleSchema> {
-    getRefreshRate: () => Promise<Milliseconds>;
-    getLastFetch: () => Promise<Timestamp>;
-    getNextFetch: () => Promise<Timestamp>;
+  getRefreshRate: () => Promise<Milliseconds>;
+  getLastFetch: () => Promise<Timestamp>;
+  getNextFetch: () => Promise<Timestamp>;
 }
 
 const moodleSchema = new Schema({
-    // Note that Moodle stores its timestamps in seconds, not in ms!
-    lastFetch: { type: Number, default: 0},
-    refreshRate: { type: Number, default: 900000},
-    nextFetch: { type: Number, default: 0},
+  // Note that Moodle stores its timestamps in seconds, not in ms!
+  lastFetch: { type: Number, default: 0 },
+  refreshRate: { type: Number, default: 900000 },
+  nextFetch: { type: Number, default: 0 },
 });
 
 moodleSchema.statics.getRefreshRate = async () => {
-    const moodle = await MoodleSettings.findOne();
-    if (!moodle || !moodle.refreshRate) {
-        loggerFile.error('MoodleSettings.getRefreshRate - could not find refresh rate, used .env values');
-        return config.moodle.fetchInterval;
-    }
+  const moodle = await MoodleSettings.findOne();
+  if (!moodle || !moodle.refreshRate) {
```

Comment:
```suggestion
  if (!moodle?.refreshRate) {
```
Why not simplify this condition?

Reply:
As I am not 100% sure that this covered by a uni test, I will reject this recommendation because it is not inscope of this pr. I will keep this in mind for the next implementation